[DE] Add `Guides\Tablet_Purchase` (#6680)
ppy/osu-wiki: wiki/Guides/Tablet_Purchase/de.md

# Ratgeber für den Kauf von Tablets

> Wenn du osu! gespielt und mit anderen Spielern gechattet hast, dann hast du bestimmt schon etwas von den sogenannten "Tablets" sowie ihre Effekte auf den Spielverlauf in osu! gehört. Tablets sind Unterlagen, die an deinen Computer angeschlossen und über einen Stift gesteuert werden. Während du den Stift über die Unterlage ziehst, bewegt sich der Mauszeiger mit dem Stift und beim Antippen wird die Maus geklickt. Du kannst dir Tablets für osu! wie einen Arcade-Stick für Kampfspiele vorstellen. Keine Voraussetzung, aber manche Spieler bevorzugen es.

Reviewer: ```suggestion
Wenn du osu! gespielt und mit anderen Spielern gechattet hast, dann hast du bestimmt schon etwas von den sogenannten "Tablets" sowie ihren Effekten auf den Spielverlauf in osu! gehört. Tablets sind Unterlagen, die an deinen Computer angeschlossen und über einen Stift gesteuert werden. Während du den Stift über die Unterlage ziehst, bewegt sich der Mauszeiger mit dem Stift und beim Antippen wird die Maus geklickt. Du kannst dir Tablets für osu! wie einen Arcade-Stick für Kampfspiele vorstellen. Keine Voraussetzung, aber manche Spieler bevorzugen es.
```

Das ist im Grunde die "Auflösung" deines Tablets oder wie groß die Bewegung sein muss, damit diese vom Tablet erkannt wird. Wenn ein Tablet 100 LPI hat, dann musst du dich mindestens 1/100tel eines Zolls bewegen, damit die Bewegung registriert wird. Jedoch sollten sogar die kleinsten LPI-Werte weitaus größer sein, als jeder Monitor, auf dem du vermutlich spielen wirst. Daher werden höhere Werte meistens einen nicht sichtbaren Effekt auf den Spielverlauf haben.

Kurz gesagt, Tablets sind ein optionales Gerät, mit dem du das Spiel auf eine andere Art und Weise spielen kannst. Sie waren ursprünglich für digitale Kunst gedacht, funktionieren aber genauso gut für osu!. Die meisten Funktionen machen keinen Unterschied in osu!.

## Tablet-Verkäufer

### osu!store

*Hinweis: **Der osu!store verkauft keine osu!tablets mehr.*** Zu dem Zeitpunkt [twitterte peppy](https://twitter.com/ppy/status/846190638873767936), dass man nach anderen Alternativen Ausschau halten sollte, sofern man diese Eingabemethode ausprobieren möchte.

> Mit dem Ziel, den Spielern zu erlauben, ein Tablet auszuprobieren, ohne viel Geld zahlen zu müssen, bietete der osu!store einst das osu!tablet an. Es wurde von Huion hergestellt und von flyte entworfen. Die erste Version im Jahr 2013 umfasste das osu!tablet (weiß), ein Stift mit einer Spitze und 3 austauschbaren Spitzen. Allerdings wurde eine AA Batterie für den Stift benötigt (diese war enthalten, machte den Stift aber schwerer). Die zweite Version im Jahr 2016 enthielt das osu!tablet (schwarz), ein Stift mit einer Spitze und 3 austauschbaren Spitzen. Hier musste der Stift via USB (enthalten) geladen werden.

Reviewer: ```suggestion
Mit dem Ziel, den Spielern zu erlauben, ein Tablet auszuprobieren, ohne viel Geld zahlen zu müssen, bot der osu!store einst das osu!tablet an. Es wurde von Huion hergestellt und von flyte entworfen. Die erste Version im Jahr 2013 umfasste das osu!tablet (weiß), ein Stift mit einer Spitze und 3 austauschbaren Spitzen. Allerdings wurde eine AA-Batterie für den Stift benötigt (diese war enthalten, machte den Stift aber schwerer). Die zweite Version im Jahr 2016 enthielt das osu!tablet (schwarz), ein Stift mit einer Spitze und 3 austauschbaren Spitzen. Hier musste der Stift via USB (enthalten) geladen werden.
```

Das HUION H430P (4096) ist vergleichbar mit dem osu!tablet (tatsächlich wird es an osu!-Spieler vermarktet).

### Wacom

Wacom ist der "Markenname" für Tablets. Tablets sind alles, was sie herstellen und sie sind weithin akzeptiert (von Künstlern) als eine Firma, die die beste Qualität sowie die zuverlässigsten Tablets bietet. Einige würden den Kauf bei Wacom empfehlen, da das Unternehmen als Marktführer beim Verkauf von Tablets gilt. Sie produzieren 3 Hauptlinien:

- Cintiq - Tablets zum "direkt auf den Bildschirm malen". Sie werden in professionellen High-End-Anwendungen eingesetzt und kosten ein hübsches Sümmchen.
- Intuos - Die professionelle Linie von Tablets. Diese haben Funktionen weit über die Bamboo-Linie hinaus, aber haben keinen Bildschirm.
- Bamboo - Die generelle Linie von Tablets für Verbraucher. Diese sind günstig und haben alle Funktionen, die du brauchst, um als Hobbyist zu starten.

Es gibt andere Linien von Wacom, wie z. B. die Tablets Graphire oder Volito, aber diese werden nicht weitergeführt und wurden von den oben genannten Linien ersetzt.

> Für osu! wirst du wahrscheinlich kein Cintiq oder Intuos benötigen, da diese beiden Linien für professionelle Arbeiten im Grafikdesign oder in der Illustration gedacht sind und vielmehr kosten als die Tablets in der Bamboo-Linie. Sie haben eine höhere Drucksensivität, mehr gebündelte Grafiksoftware und fortgeschrittene Funktionen, wie Drehung- oder Neigungssensivität. Diese kommen in osu! nicht zur Bedeutung, also kaufe dir diese nur, wenn du dein Tablet neben osu! auch für seriöses Grafikdesign verwenden möchtest.

Reviewer: ```suggestion
Für osu! wirst du wahrscheinlich kein Cintiq oder Intuos benötigen, da diese beiden Linien für professionelle Arbeiten im Grafikdesign oder in der Illustration gedacht sind und vielmehr kosten als die Tablets in der Bamboo-Linie. Sie haben eine höhere Drucksensivität, mehr gebündelte Grafiksoftware und fortgeschrittene Funktionen, wie Drehung- oder Neigungssensivität. Diese kommen in osu! nicht zur Geltung, also kaufe dir diese nur, wenn du dein Tablet neben osu! auch für seriöses Grafikdesign verwenden möchtest.
```

> Für osu! wirst du wahrscheinlich kein Cintiq oder Intuos benötigen, da diese beiden Reihen für professionelle Arbeiten im Grafikdesign oder in der Illustration gedacht sind und vielmehr kosten als die Tablets in der Bamboo-Reihe. Sie haben eine höhere Drucksensivität, mehr gebündelte Grafiksoftware und fortgeschrittene Funktionen, wie Drehung- oder Neigungssensivität. Diese kommen in osu! nicht zur Geltung, also kaufe dir diese nur, wenn du dein Tablet neben osu! auch für seriöses Grafikdesign verwenden möchtest.

Reviewer: ```suggestion
Für osu! wirst du wahrscheinlich kein Cintiq oder Intuos benötigen, da diese beiden Reihen für professionelle Arbeiten im Grafikdesign oder in der Illustration gedacht sind und vielmehr kosten als die Tablets in der Bamboo-Reihe. Sie haben eine höhere Drucksensitivität, mehr gebündelte Grafiksoftware und fortgeschrittene Funktionen, wie Drehung- oder Neigungssensitivität. Diese kommen in osu! nicht zur Geltung, also kaufe dir diese nur, wenn du dein Tablet neben osu! auch für seriöses Grafikdesign verwenden möchtest.
```

> Wacom ist der "Markenname" für Tablets. Tablets sind alles, was sie herstellen und sie sind weithin akzeptiert (von Künstlern) als eine Firma, die die beste Qualität sowie die zuverlässigsten Tablets bietet. Einige würden den Kauf bei Wacom empfehlen, da das Unternehmen als Marktführer beim Verkauf von Tablets gilt. Sie produzieren 3 Hauptlinien:
> 
> - Cintiq - Tablets zum "direkt auf den Bildschirm malen". Sie werden in professionellen High-End-Anwendungen eingesetzt und kosten ein hübsches Sümmchen.
> - Intuos - Die professionelle Linie von Tablets. Diese haben Funktionen weit über die Bamboo-Linie hinaus, aber haben keinen Bildschirm.
> - Bamboo - Die generelle Linie von Tablets für Verbraucher. Diese sind günstig und haben alle Funktionen, die du brauchst, um als Hobbyist zu starten.
> 
> Es gibt andere Linien von Wacom, wie z. B. die Tablets Graphire oder Volito, aber diese werden nicht weitergeführt und wurden von den oben genannten Linien ersetzt.
> 
> Für osu! wirst du wahrscheinlich kein Cintiq oder Intuos benötigen, da diese beiden Linien für professionelle Arbeiten im Grafikdesign oder in der Illustration gedacht sind und vielmehr kosten als die Tablets in der Bamboo-Linie. Sie haben eine höhere Drucksensivität, mehr gebündelte Grafiksoftware und fortgeschrittene Funktionen, wie Drehung- oder Neigungssensivität. Diese kommen in osu! nicht zur Bedeutung, also kaufe dir diese nur, wenn du dein Tablet neben osu! auch für seriöses Grafikdesign verwenden möchtest.
> 
> Zum Zeitpunkt der Erstellung dieses Artikels gibt es 3 Versionen von Bamboo, wobei alle verwirrend unter dem Spitznamen "Bamboo" aufgelistet werden (Die Intuos-Linien werden durch Zahlen unterschieden, Intuos5 ist der neueste aktuell). Einige würden empfehlen, entweder die zweite oder dritte Generation von Bamboo zu kaufen. Die erste hatte die zusätzlichen Buttons am oberen Rand des Tablets anstatt an der Seite, was die Benutzung in osu! erschwert.

Reviewer: ```suggestion
Wacom ist der "Markenname" für Tablets. Tablets sind alles, was sie herstellen und sie sind weithin akzeptiert (von Künstlern) als eine Firma, die die beste Qualität sowie die zuverlässigsten Tablets bietet. Einige würden den Kauf bei Wacom empfehlen, da das Unternehmen als Marktführer beim Verkauf von Tablets gilt. Sie produzieren 3 Hauptreihen:

- Cintiq - Tablets zum "direkt auf den Bildschirm malen". Sie werden in professionellen High-End-Anwendungen eingesetzt und kosten ein hübsches Sümmchen.
- Intuos - Die professionelle Reihe von Tablets. Diese haben Funktionen weit über die Bamboo-Reihe hinaus, aber haben keinen Bildschirm.
- Bamboo - Die Reihe von Tablets für allgemeine Verbraucher. Diese sind günstig und haben alle Funktionen, die du brauchst, um als Hobbyist zu starten.

Es gibt andere Reihen von Wacom, wie z. B. die Tablets Graphire oder Volito, aber diese werden nicht weitergeführt und wurden von den oben genannten Reihen ersetzt.

Für osu! wirst du wahrscheinlich kein Cintiq oder Intuos benötigen, da diese beiden Reihen für professionelle Arbeiten im Grafikdesign oder in der Illustration gedacht sind und vielmehr kosten als die Tablets in der Bamboo-Reihe. Sie haben eine höhere Drucksensivität, mehr gebündelte Grafiksoftware und fortgeschrittene Funktionen, wie Drehung- oder Neigungssensivität. Diese kommen in osu! nicht zur Bedeutung, also kaufe dir diese nur, wenn du dein Tablet neben osu! auch für seriöses Grafikdesign verwenden möchtest.

Zum Zeitpunkt der Erstellung dieses Artikels gibt es 3 Versionen von Bamboo, wobei alle verwirrend unter dem Spitznamen "Bamboo" aufgelistet werden (Die Intuos-Reihen werden durch Zahlen unterschieden, Intuos5 ist der neueste aktuell). Einige würden empfehlen, entweder die zweite oder dritte Generation von Bamboo zu kaufen. Die erste hatte die zusätzlichen Buttons am oberen Rand des Tablets anstatt an der Seite, was die Benutzung in osu! erschwert.
```
Maybe `Reihe` works better instead of `Linie`

Author: GitHub didn't like that there were several suggestions for one line. I will apply this one manually 😐

> Zum Zeitpunkt der Erstellung dieses Artikels gibt es 3 Versionen von Bamboo, wobei alle verwirrend unter dem Spitznamen "Bamboo" aufgelistet werden (Die Intuos-Linien werden durch Zahlen unterschieden, Intuos5 ist der neueste aktuell). Einige würden empfehlen, entweder die zweite oder dritte Generation von Bamboo zu kaufen. Die erste hatte die zusätzlichen Buttons am oberen Rand des Tablets anstatt an der Seite, was die Benutzung in osu! erschwert.

Reviewer: ```suggestion
Zum Zeitpunkt der Erstellung dieses Artikels gibt es 3 Versionen von Bamboo, wobei alle verwirrenderweise unter dem Spitznamen "Bamboo" aufgelistet werden (Die Intuos-Linien werden durch Zahlen unterschieden, Intuos5 ist aktuell die neueste). Einige würden empfehlen, entweder die zweite oder dritte Generation von Bamboo zu kaufen. Die erste hatte die zusätzlichen Buttons am oberen Rand des Tablets anstatt an der Seite, was die Benutzung in osu! erschwert.
```

![](img/gen1.jpg "Erste Generation Bamboo (Pen und Touch, Touch und Fun - Bamboo Fun gezeigt)")

![](img/gen2.jpg "Zweite Generation Bamboo (Pen und Touch, Pen, Touch, Fun und Craft - Bamboo Pen und Touch gezeigt)")

![](img/gen3.jpg "Dritte Generation Bamboo (Connect, Capture, Create - Connect gezeigt)")

Du solltest dich von den Banboo Touch Tablets fernhalten, da diese keine Stifteingabe unterstützen. Sie sind einfach Multitouch-Trackpads für deinen Computer. Offen gesagt [hat Apple es besser gemacht](https://www.apple.com/shop/product/MK2D3AM/A/magic-trackpad).

Die anderen Modelle von Bamboo gibt es in zwei Größen, klein und groß. In der zweiten Generation waren Bamboo Touch, Pen sowie Pen und Touch die "kleinen", wohingegen Fun und Craft die "großen" waren. In der dritten Generation sind Connect und Capture die "kleinen", Create ist "groß".

> Für osu! wirst du wahrscheinlich die "kleinen" Tablets benötigen, da es eine häufige Strategie ist, den Bereich zum Spielen zu reduzieren. Dadurch musst du deinen Arm während dem Spielen weniger bewegen. Jedoch ist das eine personelle Präferenz, ob du ein größeres Tablet möchtest oder nicht. Es lohnt sich allerdings, wenn du dich mit digitalem Zeichnen beschäftigst. Zeichnen ist viel angenehmer auf einem größeren Tablet.

Reviewer: ```suggestion
Für osu! wirst du wahrscheinlich die "kleinen" Tablets benötigen, da es eine häufige Strategie ist, den Bereich zum Spielen zu reduzieren. Dadurch musst du deinen Arm während dem Spielen weniger bewegen. Jedoch ist das eine persönliche Präferenz, ob du ein größeres Tablet möchtest oder nicht. Es lohnt sich allerdings, wenn du dich mit digitalem Zeichnen beschäftigst. Zeichnen ist viel angenehmer auf einem größeren Tablet.
```

> Und zuletzt werden CTL-480 Tablets günstig auf Second-Hand-Seiten wie eBay oder Amazon angeboten, auch wenn sie oft schwer zu finden sind, da die Produktion eingestellt wurde. Sie sind definitive eine Überlegung wert, wenn du eins für einen gerechtfertigten Preis gefunden hast, insbesondere, wenn du ein digitaler Künstler bist. Neben Wacom CTL-480, welches von den Spielern häufig genutzt und viel gelobt wird, gibt es auch die CTL-470 Aufstellung (wie die Modelle 470, 471 und 472) und die neueste CTL-4100 Serie, die die gleiche Qualität hat, wenn nicht sogar besser als die von CTL-480.

Reviewer: ```suggestion
Und zuletzt werden CTL-480 Tablets günstig auf Second-Hand-Seiten wie eBay oder Amazon angeboten, auch wenn sie oft schwer zu finden sind, da die Produktion eingestellt wurde. Sie sind definitiv eine Überlegung wert, wenn du eins für einen gerechtfertigten Preis gefunden hast, insbesondere, wenn du ein digitaler Künstler bist. Neben Wacom CTL-480, welches von den Spielern häufig genutzt und viel gelobt wird, gibt es auch die CTL-470 Aufstellung (wie die Modelle 470, 471 und 472) und die neueste CTL-4100 Serie, die die gleiche Qualität hat, wenn nicht sogar besser als die von CTL-480.
```

> Für osu! sind die Unterschiede zwischen den Intuos- und Bamboo-Linien vernachlässigbar. Der einzige wirkliche Unterschied ist, dass die Intuos-Linien eine größere Auswahl an Tabletgrößen haben als Bamboo.

Reviewer: ```suggestion
Für osu! sind die Unterschiede zwischen den Intuos- und Bamboo-Reihen vernachlässigbar. Der einzige wirkliche Unterschied ist, dass die Intuos-Reihen eine größere Auswahl an Tabletgrößen haben als Bamboo.
```
